src/main/java/de/qabel/desktop/event/SubjectEventDispatcher.java

```diff
@@ -0,0 +1,19 @@
+package de.qabel.desktop.event;
+
+import rx.Observable;
+import rx.subjects.PublishSubject;
+import rx.subjects.Subject;
+
+public class SubjectEventDispatcher implements EventDispatcher {
+    private Subject<Event, Event> subject = PublishSubject.create();
```

Comment:
better use a [SerializedSubject](http://reactivex.io/RxJava/javadoc/rx/subjects/SerializedSubject.html) to automatically handle different pushing threads